benches/gkr_hashes.rs

```diff
@@ -66,9 +84,9 @@ fn criterion_gkr_keccak(c: &mut Criterion) {
 
 fn criterion_gkr_poseidon(c: &mut Criterion) {
```

Comment:
https://github.com/PolyhedraZK/Expander/actions/runs/11076085794/job/30933755923

looks like rust ECC doesn't generate Poseidon circuit. The old Poseidon circuit triggered failure